packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -9,6 +9,12 @@ const ADVANCED_GROUP = 'advanced';
 const DISPLAY_GROUP = 'stats';
 const ZERO_CONFIG_GROUP = 'zero-config';
 
+function acceptedString(arr) {
+    return `Accepted Value: ${arr.join(' | ')}`;
+}
+function descriptionGenerator(example, shortDesc, acceptedValue = []) {
+    return `\n\n    Example: ${example}\n\n     ${acceptedValue.length > 0 ? acceptedString(acceptedValue) : ''}\n\n${shortDesc}`;
+}
```

Comment:
It was fix, not new feature, do not mix fixes and feature, you need to do it in difference PRs

Reply:
Okay. On it...